src/google-maps/deprecated-map-marker-clusterer/deprecated-map-marker-clusterer.ts

```diff
@@ -352,103 +354,143 @@ export class DeprecatedMapMarkerClusterer
   }
 
   fitMapToMarkers(padding: number | google.maps.Padding) {
-    this._assertInitialized();
```

Comment:
The whole point of these methods was that we shouldn't have to add non-null assertions everywhere. I don't think it's a bit deal that we don't tree shake the function itself, because error message is already dropped in production.